CWP/papers/HSF-CWP-2017-01_roadmap/latex/cwp-main.tex

```diff
@@ -227,7 +227,8 @@ \section{Software and Computing
 data management software to manage its share of WLCG resources.
 
 In order to process the data, the 4 largest LHC experiments have written
-more than 20 million lines of program code over the last 15 years. This
+more than 20 million lines of program code over the last 15 years
+\cite{1742-6596-898-7-072013}. This
```

Comment:
@graeme-a-stewart, I talked to Stefan Roiser to get the equivalent number for LHCb but unfortunately he does not know of a paper or even presentation that we can cite. Too bad.

Now, if the citation we have is only for ATLAS, then it might be best to have a footnote at this place where you added the citation. The footnote would then say something like "For the sake of example, the ATLAS collaboration software stack contains approximately XX million lines of code~\cite{1742-6596-898-7-072013}.". What do you think?

Reply:
I was hoping that other people would find references for the other LHC experiments. But if we only have ATLAS then maybe your suggestion makes sense.

Reply:
I doubt we get to 20M from 4 experiments. but we had agreed to make this more vague regardless, no? Its on my list to find a reference from cms (which can be GitHub if all else fails).

Reply:
From LHCb here is what I got from Stefan (no citable source, though, as said above):

> We have ~ 6 Mio SLOC of C++ & Python plus ~ 1 MIo XML, the stats from end of last year are
> 
> C++                                  3455801
> Python                               2252476
> XML                                  1062829
> C/C++ Header                       531138

Reply:
@davidlange6 - I was wondering if we might manage it, but I will update the text to be "tens of millions" in any case

Reply:
I agree with @graeme-a-stewart / @davidlange6 proposal + @eduardo-rodrigues about the footnote. Seems the way to go.

Reply:
Well, but now I added the citation to Pete's talk that @davidlange6 dug out, so there are two citations an probably better handled in the standard way